bin/adam-pyspark

```diff
@@ -23,10 +23,6 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 ADAM_JARS=$("$ADAM_REPO"/bin/compute-adam-jars.sh)
 
-if [ ! -z "$ADDL_JARS" ]; then
-  ADAM_JARS="$ADAM_JARS","$ADDL_JARS"
-fi
```

Comment:
I think it would actually become

```
./bin/adam-shell --conf spark.halt-and-catch-fire=false -- transform myBamFile.bam myAdamFile.adam
```

Note the `--`.